vendor/bat-native-ads/src/bat/ads/internal/ad_pacing/ad_pacing_test.cc

```diff
@@ -44,10 +44,10 @@ class BatAdsAdPacingTest : public UnitTestBase {
   ~BatAdsAdPacingTest() override = default;
 
   void SetUp() override {
-    UnitTestBase::SetUpForTesting(/* integration_test */ true);
+    DCHECK(CopyFileFromTestPathToTempDir(
```

Comment:
Maybe ASSERT_TRUE is better here, because DCHECK can be turned off on build agents, so instruction wouldn't be executed.

Reply:
Fixed